lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -1473,8 +1489,49 @@ sub format_id {
     return $id;
 }
 
+# Return the properly formatted JSON value
+# for the maintainer column
+sub format_maintainer {
+    my ( $c, $value, $ia, $commit ) = @_;
+    
```

Comment:
Could to do `my $user = ...` to save a call to `rs` as you did with `$complat_user`.